### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jwine8**

### CODE REVIEW

**Review by Maraudier — APPROVED** [review]
I am Matt Davis and I approve this messge

**Review by srwight — CHANGES_REQUESTED** [review]
Please include a docstring. This will go directly under your function declaration:
"""This is a docstring"""

**Review by Maraudier — CHANGES_REQUESTED** [review]
elif (a==2):
                ^
IndentationError: unindent does not match any outer indentation level

File: winegardner.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def Winegardner(a:int) -> str:
+   if (a ==1):
+        return "This is the 1st string."
+    elif (a==2):
+         return "This is the 2nd string."
+    else:
+
+    Return
+
+    return return "You passed something else." % a
```
Comment (kylestormcloud) [inline winegardner.py:10]: What is happening here?

**Pull request closed without merging**